zio-http/src/test/scala/zhttp/service/ServerSpec.scala

```diff
@@ -72,6 +81,10 @@ object ServerSpec extends HttpRunnableSpec(8088) {
             val res = app.requestBodyAsString(content = "")
             assertM(res)(equalTo(""))
           } +
+          testM("empty raw body") {
+            val res = app.requestRawBody(content = "")
+            assertM(res)(isEmpty)
+          } +
```

Comment:
I think one suite should be enough where the content is generated using a `Gen` and then asserted once.

```scala
suite("getBody") {
  test("is valid") {
    check(Gen.anyString) { string =>
       // create an app, execute and asssert
    }

  }

}
```